testcafe/e2e.spec.js

```diff
@@ -34,3 +34,46 @@ test('Captured events are accessible via /api/event', async (t) => {
     await t.expect(results.filter(({ event }) => event === 'custom-event').length).gte(1)
     await t.expect(results.filter(({ event }) => event === '$pageview').length).gte(1)
 })
+
+test('Config option mask_all_text does not set $el_text for autocaptured events', async (t) => {
```

Comment:
We're testing a negative of something that's not tested at all here.

To bring most value to the E2E suite, how about:
- One test (or extending the existing test) confirming that autocapture works as expected (e.g. captures attributes and text and whatelse)
- One test for _some_ of the negative conditions.

Generally it's really hard to test all cases E2E and these tests can be brittle - better to test these via unit tests. However autocapture is a cruicial area which we hadn't changed until now and had no e2e test coverage.

Reply:
I can do that, yeah